enhancements/cluster-logging/forwarder-label-selector.md

```diff
@@ -22,7 +22,7 @@ status: provisional
 
 ## Summary
 
-Add an input selector to the ClusterLogForwarder (CLF) to forward application
+Add an input selector to the ClusterLogForwarder (CLF) to collect application
```

Comment:
Shouldn't that be still "forward" instead of "collect"? We will not "exclude" anything at the collector level right?

Reply:
From the users point of view, we _are_ excluding logs from being collected by that input - no pipeline connected to that input will see those logs. 
What we are doing under the covers is not the users business and could change in future - you're correct that presently we collect everything and filter inside fluetnd, but it would be a great optimization to avoid reading some log files altogether if we know their contents will be completely filtered out before we get to forwarding.